### DISCUSSION TITLE
Feature Request: Option to Exclude Specific External Displays from MonitorControl

### DISCUSSION THREAD

**Discussion by KentaroxKondo** [MonitorControl/MonitorControl#1604] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

Yes, the problem occurs when I connect my Mac to a projector for presentations. I use MonitorControl to adjust the brightness of my external monitor when working from home. However, during presentations, I want to reduce the brightness of my Mac’s display to save battery, while keeping the projector’s brightness unaffected. Unfortunately, MonitorControl automatically applies its settings to the projector, which results in the audience’s view being dimmed when I lower my Mac’s brightness.

### Describe the solution you'd like

I would like to have an option to exclude specific external displays from being controlled by MonitorControl. This feature would allow users to pre-register certain monitors or projectors so that MonitorControl does not automatically adjust their settings when they are connected.

### Describe alternatives you've considered

I have considered manually turning off the MonitorControl feature whenever I use a projector. However, doing this each time is inconvenient.

**Comment by waydabber** [comment]
MonitorControl can I think control displays separately already. Or maybe I don't understand the issue. :)

**Reply by KentaroxKondo** [reply]
Can I configure MonitorControl so that the brightness doesn’t sync with my Mac’s display for specific external outputs? If so, how can I do that?
